README.md

```diff
@@ -13,6 +13,19 @@
 - [GoogleTest](https://github.com/google/googletest) (optional; bundled) - unit testing
 - Doxygen (optional) - documentation
 
+## How to install DLA-Future with spack
+
+We provide a spack package DLA-Future that can be easily add to your own spack as follows:
+
+`spack repo add spack`
+
+This will add a new repository with namespace `dlaf`.
+
+The command to install the library for example on an HPC cluster will be similar to:
+
+`spack install -v --test=root dla-future ^mkl ^hpx max_cpu_count=256`
```

Comment:
I would rephrase this to just :

Example installation: `spack install -v dla-future ^intel-mkl`.

Note that `^mkl` doesn't work because `mkl` is a virtual package.

Reply:
I didn't get any notification about your comments while I was commenting. We commented on the same things.

About this, in addition to the virtual dependency problem, I quickly thought about having the simplest form `spack install dla-future`. The problem with this, IMHO, is that hpx `max_cpu_count` option is quite important for making it work. At the same time, I know that it should be up to HPX.

Reply:
Your both right, sorry. I tried yesterday and notice that it doesn't work since, as you said, it's only a virtual package, but I forgot to fix it here. Thanks. 

Regarding the max_cpu_count I have added another line with a small explanation below. What do you think?

Reply:
LGTM